scripts/test-helm.sh

```diff
@@ -1,164 +0,0 @@
-#!/usr/bin/env bash
```

Comment:
substitute for this file?(used in prow tests)

Reply:
Just added. Good callout